### PR TITLE
Fixed issue #9995:  Internal Server Error when installing outside Doc…

### PR DESCRIPTION
**Pull request by gmambro**
…umentRoot

### CODE REVIEW

File: application/config/internal.php [hunk 1]
```diff
@@ -37,6 +37,7 @@
             'combineCss'=>false,
             'compressCss'=>false,
             'combineJs'=>$userConfig['config']['debug']>0?false:true,
+            'basePath'=> dirname(dirname(dirname(dirname(__FILE__)))),
```
Comment (olleharstedt) [inline application/config/internal.php]: What is this, really?

Comment (gmambro) [inline application/config/internal.php]: Default doesn't work when installing outside document root https://github.com/LimeSurvey/LimeSurvey/blob/83eb591e7956bdc4d6d947be4543688296af0b95/application/extensions/ExtendedClientScript/ExtendedClientScript.php#L205

**Comment by Shnoulle** [comment]
Hi,

Yes, this fix the issue BUT : i think we don't have to fix #9995 . Because here : it's really a misconfiguration. Set it "by default" seems a bad idea. 

This can be done without updating LS core : in config.php : you can set 'basePath'=>"what you want".

**Comment by gmambro** [comment]
makes sense, however documentation should be updated for this case since using alias in web server is not a misconfiguration! :D

**Pull request closed without merging**